### DISCUSSION TITLE
Live Stream Keeps Turning On

### DISCUSSION THREAD

**Discussion by AngeloAna** [tsightler/ring-mqtt#816] [Support Q&A] [answered]
I just installed the ring-mqtt add-on. I noticed when I went into the MQTT Integration that Live Stream was turned on. It could be because it is windy today and motion is being detected, even though the Ring camera itself is not showing active motion. When I turn it off, it immediately comes back on. I can see from the RTSP stream that live video is being pulled down from the cloud. How do I turn off Live Stream so I can turn it on manually when I need to? There is nothing currently requesting an RTSP stream. which I read that it would turn on Live Stream.

**Comment by tsightler** [comment]
Please carefully read all documentation and follow all advice as everything that could possibly be said about this problem has already been written.  You cannot manually turn off the stream unless it has been manually started, any other attempt to manually stop the stream will not work because the stream was started on-demand.  If the stream is starting on demand, then something is requesting a stream.

**Reply by AngeloAna** [reply]
How can I tell what is requesting the stream? I am new to HA and don't know how to find it.

**Reply by tsightler** [reply, marked as answer]
It's not really easy to see where the request is coming from, but, overall, if you've setup the camera in HA, then it's almost certainly HA itself doing it.  The FAQ covers troubleshooting and there are dozens of topics about this already.

**Reply by AngeloAna** [reply]
Thanks for pointing me in the right direction.
